[#43] user 확장, 판매자 회원가입 및 서점 등록 생성

yjk5309 opened the pull request:
판매자 로그인 후 리다이렉트 수정 예정

yjk5309 committed:
[#43] user 확장, 판매자 회원가입 및 서점 등록 생성
판매자 로그인 후 리다이렉트 수정 예정

cyj199637 commented on customer_accounts/models.py:
```diff
@@ -1,8 +1,24 @@
     birth_date = models.DateField(null=True, blank=True)
     phone_number = models.CharField(max_length=150, blank=True)
     address = models.TextField(max_length=500, blank=True)
+    is_customer = models.BooleanField(default=False)
+    is_seller = models.BooleanField(default=False)
```
사용자 유형을 나타낼 때는 이렇게 boolean 형식으로 처리하지 않고
사용자 유형이라는 컬럼 하나를 두고 팀원 간의 합의 하에 각 유형 별로 의미있는 숫자나 문자열로 나타내

yjk5309 replied:
알겠습니다.

yjk5309 committed:
[#43] 피드백 반영 및 수정
models.py 구매자 테이블 삭제, 유저 컬럼 수정(관리자, 구매자, 판매자 컬럼 하나로 구분)
서점테이블로 사업자번호 insert 되게 수정
